src/riak_kv_pb_timeseries.erl

```diff
@@ -231,7 +230,30 @@ process(#tsdelreq{table = Table, key = PbCompoundKey,
                     {reply, tsdelresp, State};
                 {error, Reason} ->
                     {reply, make_rpberrresp(
-                              ?E_DELETE, io_lib:format("Failed to delete record: ~p", [Reason])),
+                              ?E_DELETE, flat_format("Failed to delete record: ~p", [Reason])),
+                     State}
+            end
+    catch error:{undef, _} ->
+            Props = riak_core_bucket:get_bucket(Table),
```

Comment:
Missing tables are not caught, when I run `riakc_ts:list_keys/3` with a non-existing table the socket disconnects and this is written to console.log
```
2015-11-16 12:31:30.454 [error] <0.3373.0> gen_fsm <0.3373.0> in state connected terminated with reason: {error,undef,[{riak_ql_ddl_helper_mod_HiOzJEBUpAoiCHYkoF6g,get_ddl,[],[]},{riak_kv_pb_timeseries,process,2,[{file,"src/riak_kv_pb_timeseries.erl"},{line,244}]},{riak_api_pb_server,process_message,4,[{file,"src/riak_api_pb_server.erl"},{line,388}]},{riak_api_pb_server,connected,2,[{file,"src/riak_api_pb_server.erl"},{line,226}]},{riak_api_pb_server,decode_buffer,2,[{file,"src/riak_api_pb_server.erl"},{line,364}]},{gen_fsm,handle_msg,7,[{file,"gen_fsm.erl"},{line,505}]},{proc_lib,init_p_do_apply,...}]}
2015-11-16 12:31:30.454 [error] <0.3373.0> CRASH REPORT Process <0.3373.0> with 0 neighbours exited with reason: {error,undef,[{riak_ql_ddl_helper_mod_HiOzJEBUpAoiCHYkoF6g,get_ddl,[],[]},{riak_kv_pb_timeseries,process,2,[{file,"src/riak_kv_pb_timeseries.erl"},{line,244}]},{riak_api_pb_server,process_message,4,[{file,"src/riak_api_pb_server.erl"},{line,388}]},{riak_api_pb_server,connected,2,[{file,"src/riak_api_pb_server.erl"},{line,226}]},{riak_api_pb_server,decode_buffer,2,[{file,"src/riak_api_pb_server.erl"},{line,364}]},{gen_fsm,handle_msg,7,[{file,"gen_fsm.erl"},{line,505}]},{proc_lib,init_p_do_apply,...}]} in gen_fsm:terminate/7 line 622
2015-11-16 12:31:30.455 [error] <0.308.0> Supervisor riak_api_pb_sup had child undefined started with {riak_api_pb_server,start_link,undefined} at <0.3373.0> exit with reason {error,undef,[{riak_ql_ddl_helper_mod_HiOzJEBUpAoiCHYkoF6g,get_ddl,[],[]},{riak_kv_pb_timeseries,process,2,[{file,"src/riak_kv_pb_timeseries.erl"},{line,244}]},{riak_api_pb_server,process_message,4,[{file,"src/riak_api_pb_server.erl"},{line,388}]},{riak_api_pb_server,connected,2,[{file,"src/riak_api_pb_server.erl"},{line,226}]},{riak_api_pb_server,decode_buffer,2,[{file,"src/riak_api_pb_server.erl"},{line,364}]},{gen_fsm,handle_msg,7,[{file,"gen_fsm.erl"},{line,505}]},{proc_lib,init_p_do_apply,...}]} in context child_terminated
```